src/src/PluginStructs/P036_data_struct.cpp

```diff
@@ -683,21 +687,20 @@ tIndividualFontSettings P036_data_struct::CalculateIndividualFontSettings(uint8_
     String log1;
 
     if (log1.reserve(140)) { // estimated
-      delay(10);             // otherwise it is may be to fast for the serial monitor
+      delay(10);             // FIXME otherwise it is maybe too fast for the serial monitor
```

Comment:
Idem
No delay() calls in log and this doesn't even loop like it is part of debug wrappers.
Also this is way too verbose and just debug-like log entries.